.npmignore

```diff
@@ -6,8 +6,6 @@
 /.github
 /.travis.yml
 /.tx
-/test
-/webpack.config.js
```

Comment:
It looks like this change partially already exists on develop now. I guess removing /tests wasn't necessary for this repo?